server/migrations/0002-comment-deleted-at.js

```diff
@@ -0,0 +1,10 @@
+import {Comment} from '/lib/documents/comment';
+
+class Migration extends Document.AddRequiredFieldsMigration {
```

Comment:
No need to have two migrations because we didn't yet deploy the first one, and also they are of the same type.